domains/pain.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "painskids",
+        "email": "painwastooshort@gmail.com"
+    },
+    "record": {
```

```suggestion
    "records": {
```
invalid Domain Structure please read -> https://docs.is-a.dev/domain-structure/